requirements/test.txt

```diff
@@ -35,3 +35,6 @@ rouge-score>=0.0.4
 bert_score==0.3.10
 transformers>=4.0
 sacrebleu>=2.0.0
+
+# image
+pytorch_msssim @ https://github.com/jorge-pessoa/pytorch-msssim/archive/refs/heads/master.zip
```

Comment:
this may be a problem as pip does not allow including git/zip sources...
in our setup we skip all these cases:
https://github.com/PyTorchLightning/metrics/blob/72724c7c34f53d78651feb8bfdd8e99888a828bc/torchmetrics/setup_tools.py#L36
which means that this package won't be installed with `pip install torchmetrics`

Reply:
I will check another `pytorch` implementation (which is registered with pip) tomorrow: see https://pypi.org/project/pytorch-msssim/

Reply:
@Borda I updated tests accordingly. Unfortunately, the implementation above does not support `normalize` input argument. Not sure how much `coverage` will drop

Reply:
Are you still using this https://github.com/jorge-pessoa/pytorch-msssim/archive/refs/heads/master.zip ?
if not pls update also requirements...

Reply:
@stancld friendly ping here :]